src/System.CommandLine/Parsing/ArgumentResult.cs

```diff
@@ -20,9 +20,11 @@ public class ArgumentResult : SymbolResult
 
         public IArgument Argument { get; }
 
+        public bool IsImplicit => Argument.HasDefaultValue && Tokens.Count == 0;
```

Comment:
Let's keep this internal. To make it public I'd want to make sure we understand its use case better and test it more directly.

Reply:
```suggestion
        internal bool IsImplicit => Argument.HasDefaultValue && Tokens.Count == 0;
```